packages/blockchain-ui/src/theme/colors.ts

```diff
@@ -32,8 +32,9 @@ export const colors = {
   grey700: "#454F5B",
   grey800: "#212B36",
   grey900: "#161C24",
-  
-  grey500Opacity16: "919EAB29", 
+
+  grey500Opacity16: "#919EAB29",
```

Comment:
Instead of creating grey500Opacity16 in colors.ts use the following function alpha(theme.colors.grey500, 0.16) inside the component.

Reply:
Done..